### ISSUE TITLE
[Nav - EOWG] Title prefix for specific issue pages

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-website-design#85]
The WAI site currently has three pages that were created to address **specific narrow issues**:
*  Explaining [why standards harmonization is essential]( https://www.w3.org/WAI/Policy/harmon)
* The relationship between [accessibility, usability, and inclusion]( https://www.w3.org/WAI/intro/usable)
* Answering "Does W3C have [mobile accessibility standards]( https://www.w3.org/WAI/mobile/)?" and telling what we're going at W3C on mobile accessibility

Please skim those pages to get an idea of their scope.

To help communicate that these are **narrow topic pages**, I propose we categorize them. This would also help with **navigation skimming**, see [prototype navigation with shortened titles](https://w3c.github.io/wai-website-components/components/preview/example-home.html)

These were brainstorms from [1 Sept telecon](https://www.w3.org/2017/09/01-eo-minutes.html#item06) and **all of them had some -1s**
- FAQ
- Fact Sheet
- White Paper
- Brief
- at a glance (we use "At a Glance" is used for different things: https://www.w3.org/WAI/WCAG20/glance/ and https://www.w3.org/WAI/intro/atag-glance )

What other brainstorms can you come up with?

**Comment by shawna-slh** [comment]
What about: "**On Point:**"

We could have one of the home page middle boxes (in 
[prototype navigation with shortened titles](https://w3c.github.io/wai-website-components/components/preview/example-home.html) currently with dummy placeholders Audit, Design, Develop, Laws & Policies) be something like:
<hr/>

# On Point
On Point articles answer questions on:
* Does W3C have mobile accessibility standards? What resources does W3C provide on [mobile accessibility]( https://www.w3.org/WAI/mobile/)?
* The relationship between [accessibility, usability, and inclusion]( https://www.w3.org/WAI/intro/usable)
*  [Why standards harmonization is essential for web accessibility]( https://www.w3.org/WAI/Policy/harmon)

<hr/>

In Context of [nav](https://w3c.github.io/wai-website-components/components/preview/example-home.html):

### Accessibility Fundamentals
- Introduction to Accessibility
- Accessibility Principles
- Perspectives Videos
- How People with Disabilities Use the Web  >
- On Point: Standards Harmonization is Essential
- On Point: Accessibility, Usability, Inclusion
### Standards/Guidelines
- [Components of Web Accessibility]
- Web Content - WCAG
- WAI-ARIA
- Authoring Tools - ATAG
- User Agents - UAAG
- Evaluation & Report - EARL
- On Point: Mobile Accessibility at W3C
- Referencing and Linking to Standards

**Comment by VivienneConway** [comment]
+1 from me

**Comment by iamjolly** [comment]
"On Point:" sounds like a good preface for the narrow topic pages. 

I just thought of a similar one: "In Focus:" 

I believe either might work and would be happy with On Point. Curious if other folks have alternatives that would accurately reflect the content being about a more narrow topic.

**Comment by yatil** [comment]
What about:

* **Focus on** Mobile Accessibility

**Comment by yatil** [comment]
“On Point” feels like it would perpetuate the myth (?) that our other resources are not on point. I’d like to avoid it. What about: “**Focus on** Mobile Accessibility” etc. instead? I do not have strong feelings and as long as we are consistent, it shouldn’t be a big problem.

(I am unsure on having them in the boxes on the front page as they are backgrounders and not hands-on information. For example I’d prefer the “Developers’ Intro to Mobile” to the Mobile FAQ for that prime position on the home page. But I don’t feel strongly.)

**Comment by nitedog** [comment]
I like "On point". It is immediately clear that it is discussing a specific point related to the overall topic.

**Comment by ghost** [comment]
+1 with narrowing the perceived scope in navigation by categorizing them.
and +1 with 'Focus on" by @yatil 
(sorry for my terse style, y'all said relevant things and I don't have anything to add to them.)

**Comment by bakkenb** [comment]
+1 to "on point"
Definition (Merriam Webster): relating very well to the subject that is being discussed.

**Comment by vmmiller** [comment]
+1 for "In Focus:" or "Focus on:"

**Comment by shawna-slh** [comment]
I appreciate the idea of "focus" -- it's good to get multiple ideas to consider.

-1 for "Focus on Standards Harmonization is Essential", "Focus on Accessibility, Usability, Inclusion", "Focus on Mobile Accessibility at W3C"
because it becomes imperative _(expressing commands, direct requests)_, that is, we are telling people: "You should focus on this.", but these things are *not* what we want most people to focus on. We want them to know about them, but not focus on them above other resources.<br/>(@yatil, I see your point about the home page box)

In Focus: ... does not have that same issue: "In Focus: Standards Harmonization is Essential", "In Focus: Accessibility, Usability, Inclusion", "In Focus: Mobile Accessibility at W3C"

I _mildly_ prefer "On Point" to "In Focus" because "focus" is a more complex word and also it relates to vision.

**Comment by James-Green** [comment]
I really think the additional words are unnecessary and add words without value.
